src/master.py

```diff
@@ -153,22 +161,17 @@ def remaster_all(new_master_stackname):
     ]
     ec2stacks = exsubdict(ec2stacks, ignore)
 
-    def sortbypname(n):
-        unknown = 9
-        porder = {
-            #'observer': 1,
-            'elife-metrics': 2,
-            'lax': 3,
-            'basebox': 4,
-            'containers': 5,
-            'elife-dashboard': 6,
-            'elife-ink': 7
-        }
-        return porder.get(n, unknown)
+    # we can optionally pass in a list of projects to target
+    # this allows us to partition up the projects and have many of these
+    # remastering efforts happening concurrently
+    if pname_list:
+        more_ignore = [p for p in ec2stacks if p not in pname_list]
+        ec2stacks = exsubdict(ec2stacks, more_ignore)
 
-    # pname_list = sorted(ec2stacks.keys(), key=sortbypname) # lets do this alphabetically
     pname_list = sorted(ec2stacks.keys()) # lets do this alphabetically
 
+    # TODO: remove any stacks without ec2 instances
```

Comment:
todo

Reply:
there is logic for this in the reporting PR I have opened. It will get done in future